### ISSUE TITLE
Error400 on console_flow oauth

### ISSUE THREAD

**Issue by acyeagle** [burnash/gspread#1032]
**Describe the bug**
Using the console_flow option for oauth opens a webpage that throws "Error 400: invalid_request".
Page lists specifc request details:
redirect_uri: urn:ietf:wg:oauth:2.0:oob

**To Reproduce**
1. gc = gspread.oauth(flow=gspread.auth.console_flow)
2. 
3. 

**Expected behavior**
Authorization code is presented in webpage.

**Code example***
import gspread
gc = gspread.oauth(flow=gspread.auth.console_flow)

**Screenshots**
![Error400](https://user-images.githubusercontent.com/56644134/163877799-7351725d-b880-45a5-a3b3-7d8b4347bc09.png)

**Environment info:**
    Operating System: Ubuntu Server 20.04
    Python version: 3.8.10
    gspread version: 5.3.2

**Comment by aiguofer** [comment]
That flow is no longer allowed: https://developers.googleblog.com/2022/02/making-oauth-flows-safer.html#disallowed-oob

You can now do something like this:

```
from google_auth_oauthlib.flow import InstalledAppFlow

flow = InstalledAppFlow.from_client_config(config, scope)

creds = flow.run_local_server(
    host="localhost",
    port=8182,
    authorization_prompt_message="Please visit this URL: {url}",
    success_message="The auth flow is complete; you may close this window.",
)
```

**Comment by lavigne958** [comment]
Hi I tried it and faced the same issue.

I have not much to do here but remove the `console_flow` I did not catch that it was about to be deprecated.

When looking in google doc about oauth alternatives it only offers the _loopback IP address_ which is our `local_server_flow` that is already working.

Unless someone suggests something better, I will make the new release with a deprecated message and in the next major release drop the flow and any associated code.
